examples/README.md

```diff
@@ -1,20 +1,9 @@
 # Examples
 
-Run examples as follows:
+To run examples:
 
-1. Navigate to this `examples` folder
+```bash
+roc run examples/hello-world/main.roc
+```
 
-    ```bash
-    cd examples
-    ```
-
-2. Run a particular example, such as Hello World:
-
-    ```bash
-    roc run hello-world/main.roc
-    ```
-
-`crates/cli_testing_examples/benchmarks/` contains some larger examples.
-
-Some examples like `crates/cli_testing_examples/benchmarks/NQueens.roc` require input after running.
-For NQueens, input 10 in the terminal and press enter.
```

Comment:
Now the examples prompt with `Please enter an integer` so this tip seems no longer necessary.